content/blog/2018-03-06-weathercan.md

```diff
@@ -0,0 +1,476 @@
+
+``` r
+MB <- eco %>%
+  st_join(., mb_spatial) %>%
+  group_by(ECODISTRIC, ECOREGION, AREA, 
+           REGION_NAM, REGION_NOM, PROV_NAME, PROV_NOM, 
+           ECOZONE, ZONE_NAME) %>%
+  summarize(n_stations = length(unique(station_id)),
+            mean_temp = mean(mean_temp, na.rm = TRUE)) %>%
+  ungroup()
+```
+
+Finally, we can plot this as a map using `mapview`. In this manner all the data associated with each ecological region has been preserved. The image in this article is static, but if you run this yourself you'll get an interactive map and you can click on any of the regions below to get more information. Note also that the number of stations (`n_stations`) is also available.
+
+``` r
+mapview(MB, zcol = "mean_temp", legend = TRUE)
+```
+
+![](../../themes/ropensci/static/img/blog-images/2018-03-06-weathercan/mapview-1.png)
+
+<center>
+<img src = "/img/blog-images/2018-03-06-weathercan/mapview-1.png" style = "width:80%">
+</center>
+<p>
+Surprisingly Churchill, MB (the north-eastern, green area) was almost balmy compared to south-western Manitoba!
+
+Combining data in general
+-------------------------
+
+I hope these examples will help guide you in the many ways in which you can integrate `weathercan` data into other data sets. There are many different types of data to integrate, but generally, the same principles apply to merging `weathercan` data as to merging all data:
+
+-   Make sure your data is summarized to the appropriate level first (i.e. don't try to merge hourly data with yearly data)
+-   Make sure you join data by the correct columns (i.e. include your index columns as well as the appropriate time/date column)
+-   Often, you'll need to make intermediate data frames in which you link weather stations to sites or observations
+-   For spatial data, make sure your two data sets have the same CRS, and consider the `st_join()` function from the [`sf` package](http://r-spatial.github.io/sf/) to join them
+-   Make it easier on yourself by using [`tidyverse`](https://tidyverse.org), specifically the packages `dplyr` and `tidyr` ([R for Data Science](http://r4ds.had.co.nz) is a great reference)
+-   Document the process by which you join your data. This makes it easy for you to keep track of what you've done and makes your work reproducible (consider using the "Knit" button in RStudio as a shortcut for making reports)
+
+Acknowledgements
+----------------
+
+Over the course of this `weathercan` journey I've had some valuable assistance. In particular, [Sam Albers](https://github.com/boshek) has been a wonderful contributor to `weathercan` on code as well as with advice and suggestions for how to take the package to the next level. rOpenSci Reviewers [Joe Thorley](https://github.com/joethorley) and [Charles T. Gray](https://github.com/softloud), and editor [Scott Chamberlain](https://github.com/sckott) supplied wonderful comments and suggestions that were greatly appreciated.
```

Comment:
consider linking "advice and suggestions" to https://github.com/ropensci/onboarding/issues/160 so people can get a real taste of the process